cfy_manager/components/restservice/scripts/create_tables_and_add_defaults.py

```diff
@@ -92,13 +93,17 @@ def _insert_manager(config):
     sm = get_storage_manager()
     ca_cert = config.get('ca_cert')
     try:
-        stored_cert = sm.get(models.Manager, None, filters=None).ca_cert
```

Comment:
this was wrong because it looked for manager with id=none, which was never found
.list[0] is the only way we have with storage_manager to get "any" item in the table